components/main_page/wikis/deadlock/main_page_layout_data.lua

```diff
@@ -0,0 +1,146 @@
+---
+-- @Liquipedia
+-- wiki=deadlock
+-- page=Module:MainPageLayout/data
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local CONTENT = {
+	updates = {
+		heading = 'Updates',
+		body = '<nowiki>\n</nowiki>{{Main Page Updates}}',
+		padding = true,
+		boxid = 1502,
+	},
+	usefulArticles = {
+		heading = 'Useful Articles',
+		body = '{{Liquipedia:Useful Articles}}',
+		padding = true,
+		boxid = 1503,
+	},
+	wantToHelp = {
+		heading = 'Want To Help?',
+		body = '{{Liquipedia:Want_to_help}}',
+		padding = true,
+		boxid = 1504,
+	},
+	transfers = {
+		heading = 'Transfers',
+		body = '{{Transfer List|limit=15}}\n<div style{{=}}"display:block; text-align:center; padding:0.5em;">\n' ..
+			'<div style{{=}}"display:inline; float:left; font-style:italic;">\'\'[[#Top|Back to top]]\'\'</div>\n' ..
+			'<div style{{=}}"display:inline; float:right;" class="plainlinks smalledit">' ..
+			'&#91;[{{FULLURL:Player Transfers/{{CURRENTYEAR}}/{{CURRENTMONTHNAME}}|action=edit}} edit]&#93;</div>\n' ..
+			'<div style{{=}}"white-space:nowrap; display:inline; margin:0 10px font-size:15px; font-style:italic;">' ..
+			'[[Portal:Transfers|See more transfers]]<span style="font-style:normal; padding:0 5px;">&#8226;</span>' ..
+			'[[Transfer query]]<span style{{=}}"font-style:normal; padding:0 5px;">&#8226;</span>' ..
+			'[[lpcommons:Special:RunQuery/Transfer|Input Form]]' ..
+			'<span style="font-style:normal; padding:0 5px;">&#8226;</span>' ..
+			'[[Portal:Rumours|Rumours]]</center></div>\n</div>',
+		boxid = 1509,
+	},
+	thisDay = {
+		heading = 'This day in Deadlock <small id="this-day-date" style = "margin-left: 5px">' ..
+			'({{#time:F}} {{Ordinal|{{#time:j}}}})</small>',
+		body = '{{Liquipedia:This day}}',
+		padding = true,
+		boxid = 1510,
+	},
+	specialEvents = {
+		noPanel = true,
+		body = '{{Liquipedia:Special Event}}',
+	},
+	filterButtons = {
+		noPanel = true,
+		body = '<div style{{=}}"width:100%;margin-bottom:8px;">' ..
+			'{{#invoke:Lua|invoke|module=FilterButtons|fn=getFromConfig}}</div>',
+	},
+	matches = {
+		heading = 'Matches',
+		body = '{{MainPageMatches}}<div style{{=}}"white-space:nowrap; display: block; margin:0 10px; ' ..
+			'font-size:15px; font-style:italic; text-align:center;">[[Liquipedia:Matches|See more matches]]</div>',
+		padding = true,
+		boxid = 1507,
+	},
+	tournaments = {
+		heading = 'Tournaments',
+		body = '{{#invoke:Lua|invoke|module=TournamentsList|fn=run|upcomingDays=120|' ..
+			'completedDays=30|filterByTierTypes=true|useExternalFilters=true}}',
+		padding = true,
+		boxid = 1508,
+	},
+}
+
+return {
+	main = {
+		{ -- Left
+			size = 6,
```

Comment:
Consider to assign more space to the right side and reduce this to e.g. 5, as otherwise (depending on the display resolution) there isn't really enough space for the tournaments list: 
![image](https://github.com/user-attachments/assets/da90344a-12a3-4436-9ec1-d43efb11d4d7)
That one could also benefit of removed padding on it's panel.

Reply:
@mbergen Which panel specifically did you mean to have removed padding?

Reply:
5 doesn't work I think, has to be a number dividable by 12 (1, 2, 3, 4, 6, 12)

Reply:
you mean 12 dividable by it ;)

Reply:
>Which panel specifically did you mean to have removed padding?

The tournaments list panel, the padding looks weird anyways since the header bars for upcoming/ongoing/concluded don't extend to the edge

Reply:
> 5 doesn't work I think, has to be a number dividable by 12 (1, 2, 3, 4, 6, 12)

Sure it does, you can choose any number between 1 and 12. You just have to see that the total of all columns you want in the row is less or equal to 12, as otherwise they are wrapped to another row.

Reply:
So I did look at it with removing the padding, however to me it then seems pretty off / fall out of the design. (Moreso noticeable on darkmode
![image](https://github.com/user-attachments/assets/1033d21b-5b7d-40be-a798-fa1186708dea)